encoder/encoder_datatype.h

```diff
@@ -155,6 +155,9 @@ typedef struct {
 	float s_offset;
 	float c_offset;
 	float filter_constant;
+	float phase_correction;
+	float sph;
+	float cph;
```

Comment:
Add a comment for each of these what state they contain.

Reply:
Is it what you expected?
https://github.com/vedderb/bldc/pull/569/commits/fa01773ab6cf8ef1ce10c7d998b24a32be6cdba6

Reply:
Yes, thank you!

Reply:
Also write that phase_correction is in degrees.

Reply:
Done
I also reverted some changes not related with this feature: https://github.com/vedderb/bldc/pull/569/commits/164bcf7052173a67eb98ed9285b1e4ddd5f634b0